src/components/SeatMap/api.test.js

```diff
@@ -0,0 +1,226 @@
+import { JetsSeatMapApiService } from './api';
+import {
+  cabin,
+  entertainment,
+  power,
+  wifi,
+  seatDetails,
+  cabinItem,
+} from './__fixtures__/seatMapApiPostDataResponse';
+import { flightDetails } from './__fixtures__/seatMapApiGetPlaneFeatures';
+import { availabilityData } from './__fixtures__/seatMapApiAvailabilityDataResponse';
+
+jest.mock('./api', () => {
+  const module = jest.requireActual('./api');
+  return {
+    ...module,
+    JetsSeatMapApiService: class extends module.JetsSeatMapApiService {
+      postData = jest.fn();
+    },
+  };
+});
+
+const api = new JetsSeatMapApiService('test', 'test', 'test');
```

Comment:
what are these 'test' parameters, could we have variables to make it easier to understand, please?